### ISSUE TITLE
AMASS dataset preprocessing error

### ISSUE THREAD

**Issue by Kitsunetic** [sanweiliti/RoHM#20]
The preprocessing script `preprocessing_amass.py` does not generate any preprocessed data for certain AMASS subsets (CNRS, EKUT, KIT).
For all data in these subsets, it consistently shows error messages like `frame rate 100.0 not suitable for dowmsampling to 30fps`, resulting in no preprocessed data being created.
Was this intended, or is it an error?

**Comment by sanweiliti** [comment]
Yes it's intended. RoHM is trained on motion with 30fps, while AMASS subsets with a fps of 100 cannot downsampled to fps of 30, so these subsets are discarded from the training data.
